apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -212,6 +215,8 @@ private void mergeBatch(long maxSeq, Iterable<RdaChange<TClaim>> changes) {
       entityManager.getTransaction().begin();
       for (RdaChange<TClaim> change : changes) {
         if (change.getType() != RdaChange.Type.DELETE) {
+          var metaData = createMetaData(change);
+          entityManager.persist(metaData);
```

Comment:
Don't we need to persist the metadata after the claim to get the MBICacheID first?

Reply:
Good thought but the `mbiRecord` would already have been populated in the claim before `mergeBatch()` is called.